chia/cmds/plotnft_funcs.py

```diff
@@ -125,7 +125,7 @@ async def pprint_pool_wallet_state(
     print(f"Owner public key: {pool_wallet_info.current.owner_pubkey}")
 
     print(
-        f"P2 singleton address (pool contract address for plotting): "
+        f"Pool contract address (use ONLY for plotting - do not send money to this address): "
```

Comment:
do not use for XCH transactions?
do not send XCH to this address?

money works though